smart_open/s3.py

```diff
@@ -127,10 +131,21 @@ def open(
         )
     else:
         assert False, 'unexpected mode: %r' % mode
-
     return fileobj
 
 
+def _get(s3_object, version=None, **kwargs):
+    try:
+        if version is not None:
+            kwargs['VersionId'] = version
+        return s3_object.get(**kwargs)
+    except botocore.client.ClientError as error:
+        raise IOError(str(error) + "\n" +
+                      '%r does not exist in the bucket %r, version_id %r'
```

Comment:
Hanging indent

Reply:
Also, please move the conditional outside the try block. That code will never throw the ClientError